components/permissions/contexts/brave_localhost_permission_context.cc

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) 2022 The Brave Authors. All rights reserved.
```

Comment:
Not sure if the PR will be blocked for this, but this should now be 2023. Same for the `.h`.